File: exercicios/1973/1973.py
n = int(input())

t = input()
t = map(int,t.split())

pos, soma, total = 0,0,0

continua = True

for i, num in enumerate(t):
    total+=num
    
    if (num % 2 == 0 and continua):
        ataques = i+1
        soma += ((i*2) -1) - pos
        continua = False

    if(num - 1 == 0 and continua):
        pos = i + 1
else:
    if (soma > 0):
        total -= soma
    else:
        ataques = n
        total -= ataques
    
print("%d %d"%(ataques,total))

File: exercicios/JavaScript/1973/1973.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var n = lines.shift();
var qtx = n
var v = lines.shift().split(' ')


var continua = true;
var nums = [];
var total = 0;
var ataques = 0;
var soma = 0;
var pos = 0;
var pJ = 0
for(let i=0;i<n;i++){
    nums[i] = parseInt(v[i]);
    total += parseInt(v[i]);
}

for(let j=0; j<n; j++){
    if((parseInt(nums[j])%2==0) && continua){
        ataques = j+1;
        soma += ((j*2)+1) - pos;
        continua = false;
    }
    if(nums[j] -1 == 0 && continua){
        pos = j + 1;
    }
}

if(soma>0){
    total -= soma;
}else{
    ataques = qtx;
    total -= ataques;
}
console.log(parseInt(ataques),parseInt(total));